src/Integration.Vsix/SonarLintDaemon/SonarLintDaemon.cs

```diff
@@ -100,17 +107,69 @@ public void Start()
             }
 
             port = TcpUtil.FindFreePort(DEFAULT_DAEMON_PORT);
-            process = new Process
+            process = new System.Diagnostics.Process
             {
                 StartInfo = new ProcessStartInfo
                 {
                     UseShellExecute = false,
                     FileName = ExePath,
                     Arguments = GetCmdArgs(port),
-                    CreateNoWindow = true
+                    CreateNoWindow = true,
+                    WindowStyle = ProcessWindowStyle.Hidden,
+                    RedirectStandardOutput = true
                 }
             };
+            CreatePane(new Guid(), "SonarLint Daemon", true, false);
+            process.OutputDataReceived += (sender, args) => pane.OutputString(args.Data + "\n");
+            pane.OutputString("Running " + ExePath + "\n");
             process.Start();
+            process.BeginOutputReadLine();
+
+
+
+            channel = new Channel($"{DAEMON_HOST}:{port}", ChannelCredentials.Insecure);
+            daemonClient = new StandaloneSonarLint.StandaloneSonarLintClient(channel);
+
+            ListenForLogs();
+        }
+
+        private async System.Threading.Tasks.Task ListenForLogs()
+        {
+            // Wait for the server to be started
+            await System.Threading.Tasks.Task.Delay(4000);
+            try
+            {
+                using (var streamLogs = daemonClient.StreamLogs(new Sonarlint.Void(), cancellationToken: channel.ShutdownToken))
+                {
+                    pane.OutputString("Log listener started\n");
+                    while (await streamLogs.ResponseStream.MoveNext())
+                    {
+                        var log = streamLogs.ResponseStream.Current;
+                        pane.OutputString(log.Level + " " + log.Log + "\n");
+                    }
+                }
+            }
+            catch (RpcException e)
```

Comment:
It actually is reachable - using is a shorthand for try-finally. It is similar to a C++ pattern of local variable in a block:
{
   Object obl;
}
// obj out of scope